src/Sulu/Bundle/SecurityBundle/Entity/Role.php

```diff
@@ -93,14 +98,9 @@ public function getId()
         return $this->id;
     }
 
-    public function getRole()
-    {
-        return 'ROLE_SULU_' . strtoupper($this->name);
-    }
-
```

Comment:
```suggestion

    /**
     * @deprecated Deprecated since Sulu 2.1 will be removed in 3.0 use "getIdentifier" instead.
     */
    public function getRole()
    {
        @trigger_error(sprintf('The "%s" method is deprecated since Sulu 2.1, use "%s" instead.', __METHOD__, 'getIdentifier'), E_USER_DEPRECATED)
        return $this->getIdentifier();
    }
```